wgpu-hal/src/lib.rs

```diff
@@ -700,20 +827,36 @@ pub trait Queue: WasmNotSendSync {
     ///
     /// Valid usage:
     ///
-    /// - All of the [`CommandBuffer`][cb]s were created from
-    ///   [`CommandEncoder`][ce]s that are associated with this queue.
+    /// - All of the [`CommandBuffer`][cb]s must have been created from
```

Comment:
- [x] issue: A definite article here is jarring, since no definite selection of a subject happened before it.

```suggestion
    /// - Each [`CommandBuffer`][cb] must have been created from
```

I know this is from older documentation, so I'm happy to iterate in my own PR, if you want.